examples/02_first_level_models/plot_fir_model.py

```diff
@@ -0,0 +1,155 @@
+    contrasts = dict([(column, contrast_matrix[i])
+                      for i, column in enumerate(design_matrix.columns)])
+
+    # Add more complex contrasts
+    contrasts['audio'] = (contrasts['clicDaudio']
+                          + contrasts['clicGaudio']
+                          + contrasts['calculaudio']
+                          + contrasts['phraseaudio']
+                          )
+    contrasts['video'] = (contrasts['clicDvideo']
+                          + contrasts['clicGvideo']
+                          + contrasts['calculvideo']
+                          + contrasts['phrasevideo']
+                          )
+    contrasts['computation'] = contrasts['calculaudio'] + contrasts['calculvideo']
+    contrasts['sentences'] = contrasts['phraseaudio'] + contrasts['phrasevideo']
+
+    # Short dictionary of more relevant contrasts
+    contrasts = {
+        'left-right': (contrasts['clicGaudio']
+                       + contrasts['clicGvideo']
+                       - contrasts['clicDaudio']
+                       - contrasts['clicDvideo']
+                       ),
+        'H-V': contrasts['damier_H'] - contrasts['damier_V'],
+        'audio-video': contrasts['audio'] - contrasts['video'],
+        'computation-sentences': (contrasts['computation'] -
+                                  contrasts['sentences']
+                                  ),
+    }
+    return contrasts
+
+#########################################################################
+# Result plotting function
+
+from nilearn import plotting
+import matplotlib.pyplot as plt
+
+def plot_contrast(first_level_model):
+    """ Given a first model, specify, enstimate and plot the main contrasts"""
+    design_matrix = first_level_model.design_matrices_[0]
+    # Call the contrast specification within the function
+    contrasts = make_localizer_contrasts(design_matrix)
+    fig = plt.figure(figsize=(11, 3))
+    # compute the per-contrast z-map
+    for index, (contrast_id, contrast_val) in enumerate(contrasts.items()):
+        ax = plt.subplot(1, len(contrasts), 1 + index)
+        z_map = first_level_model.compute_contrast(
+            contrast_val, output_type='z_score')
+        plotting.plot_stat_map(
+            z_map, display_mode='z', threshold=3.0, title=contrast_id, axes=ax,
+            cut_coords=1)
+
+
+#########################################################################
+# Next solution is to try fininte impulse reponse (FIR) models: we just say that the hrf is an arbitrary function that lags behind the stimulus onset.
+# In the present case, given that the numbers of condition is high, we should use a simple FIR model.
+# 
+# Concretely, we set `hrf_model` to 'fir' and `fir_delays` to [3, 5, 7] (s)
+
+from nistats.first_level_model import FirstLevelModel
+from nistats.reporting import plot_design_matrix, plot_contrast_matrix
+
+first_level_model = FirstLevelModel(t_r, hrf_model='fir', fir_delays=[1, 2, 3])
```

Comment:
does `fir_delays=[1, 2, 3]` correspond to 3, 5, 7s as stated above? What is the unit?